### ISSUE TITLE
Add my hosts?

### ISSUE THREAD

**Issue by dalisoft** [StevenBlack/hosts#1298]
Hi @StevenBlack 

Thanks for large database for blocking ads, trackers and more unnecessary things.

I'd like to contribute project with my own blacklist, you find it at [here](https://github.com/dalisoft/hosts)

---
ghost 
## Compare
```bash
hosts on  master 
❯ ghosts -c dalisoft-hosts
----------------------------------------
Base hosts file summary:
----------------------------------------
Location: https://raw.githubusercontent.com/StevenBlack/hosts/master/hosts
Domains: 57,398
Bytes: 1.8 MB
----------------------------------------
----------------------------------------
Compared hosts file summary:
----------------------------------------
Location: dalisoft-hosts
Domains: 167
Bytes: 5.8 kB
----------------------------------------
Intersection: 21 domains
```

**Comment by dalisoft** [comment]
I'm tracking this list within 3 month, but recently made as GitHub repo.
I'm combining your hosts + my own hosts + AdAway hosts (i know it's included in your hosts, but some of item is missing from AdAway's host) = very cool results, got ~80 balls from [here](https://checkadblock.ru)

**Comment by dalisoft** [comment]
@StevenBlack How i can add my own hosts can add to list to your hosts? Any guide please?

**Comment by StevenBlack** [comment]
Hi @dalisoft thanks for this offer.  

It's not so simple.

Firstly, we amalgamate hosts files from curators with a proven track record.  The `dalisoft-hosts` file has been around for under one month, at this juncture.  So it's hard to judge its long-term value.

Secondly the `dalisoft-hosts` treads fairly heavily on some contentious properties (Facebook, Youtube, Google, Microsoft, Apple) which have taken us a long time to balance and fine-tune.  

To be honest, I haven't taken the time (yet) to see which of those domains would be added to our amalgamated lists, which ones have been the subject of prior shitfights, etc.  You haven't fought the battles we've fought to find a good balance here.  

The best way to create an avalanche of new issues here, an enormous time sink, is to block some facet of YouTube or Facebook or Microsoft that are, or are not, related to tracking or advertising and also some subtle facet of legitimate content delivery on those platforms.

In short, the PR is fine!  It will take awhile to analyze the possible side-effects which seem potentially heavy, here.

**Comment by dalisoft** [comment]
Thanks for reply @StevenBlack 

I blocked from domains of these companies, because they are tracking me which I don’t want. 
Recently i even removed my own ad blocker due of authors did not response why they are using own ssl. I care about privacy more than ad blocking. 

Yes, you right. Project already lives long time, exactly, project not easy got current status, i too own open source project and I understand your position. 

Then, please, check out my domains list, if it is not OK, you can close PR and this issue. I understand you. 

Or there i close this issue and PR and i continue track and update my lists. 
After some times (6-12 month), i let you know, if everything is OK, we will do some action (merge or give more time). 

My english not good, i hope you understand 
Thanks again and you maintaining very useful project, i wish you best of best for this useful project. 

Regards, 
Davlik

**Comment by StevenBlack** [comment]
Thank you for understanding @dalisoft, give it time.

Your English is fine.

Some of the effects are subtle.  Do you care about accessing your Youtube viewing history, as tallied by Youtube?  I can tell you, some people certainly do.  And we hear about it when we block the Youtube subdomain that does that.

**Comment by dalisoft** [comment]
@StevenBlack Yes, sometimes i need my YouTube history. 
My hosts doesn't affects to YouTube history. 
I always test my hosts after blocking, everything works fine or not. If something broken, i will revert or remove from list. 

But let’s we do our best to maintain our hosts. 
If users need my hosts, users always can download from my repo, after some times i should get issue tickers which services doesn’t works, i will try debug and fix it. 
When my repo will be close to stable, then i let you know and we will think future actions.
